TWG/MeetingMinutes/2020-11-30-TWG-Minutes.md.md

```diff
@@ -0,0 +1,73 @@
+TWG Meeting Report
+Nov 30/2020
+
+## TG Reports
+
+### SW TG
+
+Written report prepared - thanks to Jeremy:
+https://github.com/openhwgroup/core-v-docs/blob/master/program/TG-reports-for-TWG/20201130-sw-tg.md
+
+IDE - Nearing CQ. Most important issue is "hello world application".
+
+GNU Tools - Good progress. Need to have FSF assignment from contributors - Action to Duncan. 
+FSF assignment from ETH Z needs to be requested. Rick to help out. Target board needed. 
+
+CLANG/LLVM - PPL report below
+
+FreeRTOS - PPL report below
+
+
+### HW TG
+Verbal report:
+
+CORE-V-MCU – Quicklogic is putting resources in place. Need to review against original PPL plan and schedule PL.
+
+CORE-V-VEC – Passed PPL (Nov 23). Research proposal submitted to Mitacs and feedback coming in January.
+
+CORE-V-FPGA - missed the summary - please provide, Hugh
+
+
+
+## PPL Proposals
+
+FreeRTOS 
+https://github.com/openhwgroup/core-v-docs/blob/master/program/core-v-free-rtos-ppl.md
+
+The FreeRTOS PPL proposal was prepared by a team including  Jeremy Bennett, Robert Balas, Shteryana Shopova, Olive Zhao, Duncan Bees and others.
+
+In the PPL presentation by Jeremy, he noted
+FreeRTOS is already widely used by OpenHW members and is the most popular RTOS in the world
+Zephry is more widely spoken of in RISC-V world because of AntMicro – in future, we may wish to undertake support. 
+Current issues:
+* The Pulpissimo FreeRTOS work needs to be available via the legal offfice. However, if that doesn't happen, the FreeRTOS port could start from scratch.
+* Development board needs to be supplied to Shteryana – has been sent to Bulgaria
+* Simon noted that Imperas has already supported FreeRTOS for other platforms and plan to support for OpenHW when it is running on the FPGA board. So he will be added as a supporter
+ * Q? why do we need the Imperas modelling if actual boards are available?

```

Comment:
General MarkDown formatting comment. Indents on lists are by increasing the asterisk characters, not by indenting. Thus this line should start `**`, similarly for other indented bullets.

Reply:
hmm.. saw this method on a reference but thanks.